packages/core/lib/commands/db/commands/serve.js

```diff
@@ -17,7 +17,15 @@ const command = {
    */
   run: async function (argv) {
     const Config = require("@truffle/config");
-    const { serve } = require("@truffle/db");
+    const { getTruffleDb } = require("@truffle/db-loader");
+    const Db = getTruffleDb();
```

Comment:
This shouldn't block the PR, but we should probably guard `db serve` behind the `db: enabled` config/feature flag